src/modules/addScores.js

```diff
@@ -1,56 +1,77 @@
 class LeaderBoard {
```

Comment:
Great job @Otegaa 🤝

- [ ] [Optional] I would suggest you use function components instead of class components in order to take advantage of the merits of function components over class components.